parts/linux/cloud-init/nodecustomdata.yml

```diff
@@ -432,7 +432,16 @@ write_files:
   owner: root
   content: !!binary |
     {{GetVariableProperty "cloudInitData" "containerdKubeletDropin"}}
-{{if UseRuncShimV2}}
+
+{{- if Is2204VHD}}
```

Comment:
@UtheMan look at this line, this file, 10-cgroupv2.conf will only be added to the custom data if "Is2204VHD" is true. this is what baker_test.go validates.